include/deal.II/numerics/data_out.h

```diff
@@ -516,6 +516,18 @@ class DataOut : public DataOut_DoFData<DoFHandlerType,
                   const CurvedCellRegion              curved_cell_region);
 };
 
+namespace Legacy
+{
+  /**
+   * @deprecated Use dealii::DataOut without the DoFHandlerType template
+   * instead.
+   */
+  template <int dim, typename DoFHandlerType = DoFHandler<dim>>
+  class DEAL_II_DEPRECATED DataOut : public dealii::DataOut<dim, DoFHandlerType>
+  {
+    using dealii::DataOut<dim, DoFHandlerType>::DataOut;
```

Comment:
Ah, that's another way -- but I think that the type alias is still the better choice.